### DISCUSSION TITLE
Wiring - Intelliflow to Rs-485 USB Controller

### DISCUSSION THREAD

**Discussion by fteston** [tagyoureit/nodejs-poolController#555] [RS485] [answered]
Equipment supported

Controllers: IntelliCenter, Intellitouch, EasyTouch, No controller (standalone equimpent)
**Pumps: Intelliflow VS/VSF/VF, older models**
Chlorinators: Intellichlor, Aqua-Rite and OEM brands
Heaters: Gas, solar, heatpump
Intellichem and Relay Equipment Manager (REM) chemical controllers
Intellivalve (coming soon)
Home Automation: SmartThings, Hubitat, ISY, Vera, Siri, Echo
Chemical probes (pH, ORP, flow sensors, EC, etc.)

![image](https://user-images.githubusercontent.com/5831307/180281016-8d22445d-4b4c-4b41-91af-2e65218a7f87.png)

Could I directly connect the Intelliflow port using the cable Pentair 350122 , Pentair Intelliflo Variable Speed Pump to [Moyina USB to RS485 Adapter](https://www.amazon.com/dp/B07SD65BVF/?coliid=I2ZXMUX22W8KIE&colid=151RLMN7IIZ9X&psc=1&ref_=lv_ov_lig_dp_it) and a Raspberry Pi? 

Do you have the wiring diagram to connect from Pentair 350122 (Pin 6 / Pin 7) to Rs-485 Adapter? 

![image](https://user-images.githubusercontent.com/5831307/180281733-444ea60f-363d-45d9-86b3-e22ddd1d8880.png)

Thank you,

Fernando

**Comment by tagyoureit** [comment, marked as answer]
Yes, this is one of the community recommended adapters on the [wiki](https://github.com/tagyoureit/nodejs-poolController/wiki/RS-485-Adapter-Details#step-2).

However, you should connect the adapter to the Intellicenter and then you can use the full functionality of our apps.  Your pumps should already be connected to the rest of the equipment and hence you would be disconnecting the pump from IC control.  
[EDIT] - I may have been a bit hasty - if you don't have an existing outdoor control panel this isn't a problem.  It was hard to tell from your initial post what is your equipment and what you just copy/pasted.

**Reply by rstrouse** [reply]
Yellow is typically A+ and green is typically B-.  Do not forget you need to define the pump in dashPanel before Nixie will start talking to it.

**Reply by fteston** [reply]
Hi yes I only have the pump not external controller, I understand that with nodejs-poolController is not needed to have any external controller and I could directly connect to RS-485 and a RP. 

![image](https://user-images.githubusercontent.com/5831307/180284769-fb55b99f-8b95-4622-a874-c8a5573aac7e.png)

![image](https://user-images.githubusercontent.com/5831307/180284977-fc87b992-4d7e-4224-88ed-70c1c90c93e0.png)

I will try connect the wires followin @rstrouse recommencation.

Thank you,
Fernando
